config/ember-try.js

```diff
@@ -1,6 +1,5 @@
 /*jshint node:true*/
 module.exports = {
-  useVersionCompatibility: true,
```

Comment:
@marcoow was this an intentional change? as discussed earlier today we should most likely test *all* minor Ember releases since we're using an Handlebars AST transform here and the AST is not public/stable.